superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

```diff
@@ -48,9 +48,12 @@ export default function EchartsTimeseries({
   onContextMenu,
   xValueFormatter,
   xAxis,
+  refs,
 }: TimeseriesChartTransformedProps) {
   const { emitFilter, stack } = formData;
   const echartRef = useRef<EchartsHandler | null>(null);
+  // eslint-disable-next-line no-param-reassign
+  refs.echartRef = echartRef;
```

Comment:
It seems `echartRef` is only being used internally in `EchartsTimeseries`. Do we need to add it to `refs`?

Reply:
It was passed to `Echart` in a separate dedicated `ref` field. TO avoid having two properties (one for the div ref and one for the echart ref), I decided to just place both of them in that `refs` object (I was becoming tired at this point and running low on imagination 😄 )

Reply:
Bu `Echart` is not using `echartRef`